_pages/cbio-who.md

```diff
@@ -174,8 +166,21 @@ Institute on 6 December 2023.
   [*pRoloc*](https://bioconductor.org/packages/pRoloc).
 
 
+- [**Christophe Vanderaa**](https://github.com/cvanderaa), statOmics group,
+  UGent, Belgium. We pursue our collaboration on the development of
+  computational solutions for the analysis of single-cell proteomics
+  data.
+
 ## Alumni
 
+- **Christophe Vanderaa** (ORCID:
+  [0000-0001-7443-5427](https://orcid.org/0000-0001-7443-5427),
```

Comment:
could you add your current affiliation, to illustrate what past members have moved on to doing after their time in the lab

Reply:
done :+1: